packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1094,7 +1121,11 @@ export class MergeTree {
     public localNetLength(segment: ISegment) {
         const removalInfo = toRemovalInfo(segment);
         if (removalInfo !== undefined) {
-            return 0;
+            const [removedSeq] = normalizeSequenceNumberIfUnassigned(removalInfo.removedSeq);
+            if(removedSeq > this.collabWindow.minSeq) {
+                return 0;
+            }
+            return undefined;
```

Comment:
Can you add a comment explaining why this is undefined outside the collab window?